_posts/izayak/2016-02-15-tetris-turtle-ry.md

```diff
@@ -0,0 +1,15 @@
+---
+author: izayak
+title: "Ruotong's Tetris Turtles Refactor"
+layour: post
+---
+
+<iframe src="https://trinket.io/embed/python/1c64401265" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>  
```

Comment:
Check your link URL and/or make sure you've saved in trinket.  This seems to be the original.